app/components/content/mailing_list_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="action-container">
+  <div class="heading-container">
+    <h2 class="heading-m heading--box-purple">
+      Get the latest teaching information and support
+    </h2>
+  </div>
+
+  <div class="content">
+    <div class="inset">
+      <p>
+        Answer a couple of questions about yourself and we'll send you more information about teaching
```

Comment:
@philspray Just checking we are not being too misleading here - are we sure that they will receive information about salaries on these email journeys? If that is not definite, should we just say 'the benefits of teaching' or 'teaching benefits'?